src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,5 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
It's better to reset `margin` and `padding` for the list of tags that have default values, and you don't need them